test/functional/p2p_ibd_txrelay.py

```diff
@@ -22,7 +40,19 @@ def set_test_params(self):
             ["-minrelaytxfee={}".format(NORMAL_FEE_FILTER)],
         ]
 
-    def run_test(self):
+    def test_inv_ibd(self):
+        self.log.info("Check that nodes don't request txn while still in IBD")
+        peer = self.nodes[0].add_p2p_connection(P2PInterface())
+        peer.send_message(msg_inv([CInv(t=MSG_WTX, h=0xffaa)]))
+        # As tx-announcing connection is inbound, wait for the inbound delay applied by requester.
+        sleep(NONPREF_PEER_TX_DELAY)
```

Comment:
Could speed this up? I think txrequest works with mocktime, too

Reply:
Yes I can confirm txrequests works with mocktime. Updated with your suggestion.